### ISSUE TITLE
[a11y]: form is inaccessible 

### ISSUE THREAD

**Issue by CBID2** [devleo10/TechPulse#37]
#### Problem 
I did an accessibility check and found that the search input's placeholder is uninformative. According to WC3, [a label and a form control should be associated with each other either implicitly or explicitly](https://www.w3.org/WAI/tutorials/forms/labels/). This would make it easier for users with screen readers to use the search engine.

![results of accessibility](https://github.com/user-attachments/assets/28cd7982-84cf-4f6d-9ba7-3f038517cdf3)

#### Solution 
 I will add `<label>` tags to improve the form

**Comment by AnkanMisra** [comment]
@CBID2 You are assigned for this issue

**Comment by CBID2** [comment]
Hey @AnkanMisra can you add the aperte 2.0 label?
